common-build.gradle

```diff
@@ -47,3 +47,133 @@ task syncVersionAndroid  {
     }
 }
 
+task generateIOSBindings {
+    doLast() {
+        // Only generate bindings for iOS projects
+        if ( !project.hasProperty('iosSdkModuleName') ) {
```

Comment:
I feel like we should have another ios-common-build.gradle or something instead of inspecting the name

Reply:
Done.